server/src/pages/shot/share-buttons.js

```diff
@@ -23,10 +23,9 @@ exports.ShareButton = class ShareButton extends React.Component {
         isExtInstalled={this.props.isExtInstalled}
       />;
     }
+    const active = this.state.display ? "active" : "inactive";
     return <div>
-      <button className="button primary" id="toggle-share" onClick={ this.onClick.bind(this) }>
-        <span className="flex"><img className="share-icon" src={ this.props.staticLink("/static/img/share.svg")} />Share</span>
-      </button>
+      <button className={`button secondary share ${active}`} id="toggle-share" onClick={ this.onClick.bind(this) } alt="Share" />
```

Comment:
Does the `<button>` element support `alt=""` attribute, or only `<img>`?
Should this be `title="Share"`?